src/components/Link.tsx

```diff
@@ -43,32 +50,47 @@ const Link: React.FC<LinkProps> = props => {
     underline = false,
     onClick,
     disabled,
+    href,
     ...linkProps
   } = props;
 
   const classes = useStyles(props);
 
-  return (
-    <Typography
-      component="a"
-      className={classNames(className, {
-        [classes.root]: true,
-        [classes[color]]: true,
-        [classes.underline]: underline,
-        [classes.disabled]: disabled
-      })}
-      onClick={event => {
-        if (disabled) {
-          return;
-        }
+  const commonLinkProps = {
+    className: classNames(className, {
+      [classes.root]: true,
+      [classes[color]]: true,
+      [classes.underline]: underline,
+      [classes.noUnderline]: !underline,
+      [classes.disabled]: disabled
+    }),
+    onClick: event => {
+      if (disabled) {
+        return;
+      }
 
-        event.preventDefault();
-        onClick();
-      }}
-      {...linkProps}
-    >
-      {children}
-    </Typography>
+      event.preventDefault();
+      onClick();
```

Comment:
Since `onClick` is optional I think it would be a good idea to check if the function exists

Reply:
done